File: c/practice/q1.c
# include<stdio.h>

int main(void){
    int length = 10;
    char dnaSequence[length];
    for(int index = 0; index < length; index++){
        printf("Please enter the input: ");
        scanf("%c%*c", &dnaSequence[index]);
        while(dnaSequence[index] != 'G' && dnaSequence[index] != 'A' && dnaSequence[index] != 'T' && dnaSequence[index] != 'C' ){
            printf("Please enter a correct input: ");
            scanf("%c%*c", &dnaSequence[index]);
        }
    }
    printf("Sequence \n");
    for (int index=0; index < length; index++) {
        printf("%c", dnaSequence[index]);
    }
    printf("\n");
}

File: c/practice/q1.m
wlength = 10;
dnaSequence = [length];
for i = 1 : length
   dnaSequence(i) = input("Please enter the DNA Sequence: ",'s'); 
   while (~(dnaSequence(i) == 'G' || dnaSequence(i) == 'T' || ...
           dnaSequence(i) == 'A' || dnaSequence(i) == 'C'))
       dnaSequence(i) = input("Wrong input enter again: ",'s'); 
   end
end

fprintf("Sequence\n");
for i = 1 : length
   fprintf("%c", dnaSequence(i));
end